pkg/crc/machine/errors.go

```diff
@@ -1,27 +1,17 @@
 package machine
 
 import (
-	"fmt"
-
-	"github.com/code-ready/crc/pkg/crc/logging"
+	"github.com/pkg/errors"
 )
 
 func startError(description string, err error) (*StartResult, error) {
-	fullErr := logErrorf("%s: %v", description, err)
-	return nil, fullErr
+	return nil, errors.Wrap(err, description)
```

Comment:
At this point, these helpers can be removed.

Reply:
I was tempted but it forces us to add an extra explanation for startup error. It's easy to see in Start() if an error is not using `startError`. 

I removed the helper for console and status. Errors were not bringing much info.